inception-imls-opennlp/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/opennlp/ner/OpenNlpNerRecommender.java

```diff
@@ -352,7 +352,7 @@ private NameSample createNameSample(CAS aCas, AnnotationFS aSentence,
             if (beginToken == null || endToken == null) {
                 LOG.warn("Skipping annotation not starting/ending at token boundaries: [{}-{}, {}]",
                         annotation.getBegin(), annotation.getEnd(), label);
-                continue;
```

Comment:
No, the `continue` is needed here! Without it, the loop continues to execute which is not wanted. If `beginToken` or `endToken` is null, the loop should continue with the next iteration.